state/protocol/protocol_state/kvstore/models.go

```diff
@@ -139,7 +139,7 @@ func (model *Modelv0) GetEpochExtensionViewCount() uint64 {
 	return model.EpochExtensionViewCount
 }
 
-func (model *Modelv0) GetEpochCommitSafetyThreshold() uint64 {
+func (model *Modelv0) GetFinalizationSafetyThreshold() uint64 {
 	return model.EpochCommitSafetyThreshold
```

Comment:
we are still using the old name here. Its probably a breaking change to rename this value, but maybe we could do that before Crescendo?